CHANGELOG.md

```diff
@@ -6,14 +6,17 @@ x.x.x Release notes (yyyy-MM-dd)
 ### Fixed
 * Fixed an issue in `toJSON()`, in combination with primaryKeys, where data from another table could be returned. ([#3331](https://github.com/realm/realm-js/issues/3331), since v10.0.0)
 * Fixed an issue in `toJSON()` where `data` would output as `{}`, it now returns the data base64 encoded. ([#3356](https://github.com/realm/realm-js/pull/3356), since v10.0.0)
+* Fixed a crash in case insensitive query on indexed string properties when nothing matches. ([realm/realm-cocoa#6836](https://github.com/realm/realm-cocoa/issues/6836), since v6.0.0)
```

Comment:
@fronck Be aware when copying changelogs from core to update the "since x.y.z" to the JS version where that core x.y.z version was used.

Reply:
v6.0.0 is special - JS v6.0.0 upgraded to core v6.0.0.